content/kubernetes/release-notes/k8s-6-2-10-45-2022-07.md

```diff
@@ -0,0 +1,151 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.2.10-45 (July 2022)
+linktitle: 6.2.10-45 (July 2022)
+description: Support for REBD specific upgrade policies, memcached type REDBs, and RHEL8 for OpenShift, as well as feature improvements and bug fixes.
+weight: 62
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-6-2-10-45-2022-02.md,
+    /kubernetes/release-notes/k8s-6-2-10-45-2022-02/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.10-45 supports the Redis Enterprise Software release 6.2.10 and includes feature improvements and bug fixes.
+
+The key bug fixes and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.10-129` or  `redislabs/redis:6.2.10-129.rhel8-openshift`
```

Comment:
```suggestion
* **Redis Enterprise**: `redislabs/redis:6.2.10-129` or  `redislabs/redis:6.2.10-129.rhel7-openshift`
```

Reply:
@zcahana should we only mention the rhel7 one to avoid confusion?

Reply:
I think we should try to find the least confusing way to mention both, since rhel8 is the primary image and is the desired one looking forward, and rhel7 is only there as a workaround, hopefully temporary.